### ISSUE TITLE
New button seems broken

### ISSUE THREAD

**Issue by JMante1** [SynBioHub/synbiohub.github.io#46]
The link on the 
![image](https://user-images.githubusercontent.com/40006518/100550703-e9fa8980-3238-11eb-9275-9950aa0c15bc.png) button seems broken.

**Comment by PRUBHTEJ** [comment]
Yeah actually the version for ACE-documentation was recently updated and therefore this was newly added. Hence even I don't know about the function of this button, though I'll try and find out.

**Comment by PRUBHTEJ** [comment]
Where should we link this button to, though?

**Comment by cjmyers** [comment]
@JMante1 please comment

**Comment by JMante1** [comment]
I think the button should just be removed

**Comment by PRUBHTEJ** [comment]
I'll remove this.
